saleor/graphql/app/types.py

```diff
@@ -52,13 +53,28 @@
     ThumbnailByAppInstallationIdSizeAndFormatLoader,
     app_promise_callback,
 )
-from .enums import AppExtensionMountEnum, AppExtensionTargetEnum, AppTypeEnum
+from .enums import (
+    AppExtensionMountEnum,
+    AppExtensionTargetEnum,
+    AppTypeEnum,
+    CircuitBreakerState,
+    CircuitBreakerStateEnum,
+)
 from .resolvers import (
     resolve_access_token_for_app,
     resolve_access_token_for_app_extension,
     resolve_app_extension_url,
 )
 
+# TODO: Remove the conditional when unit tests circular import is solved.
+breaker_board = None
+if settings.BREAKER_BOARD_ENABLED:
+    from ...webhook.circuit_breaker.breaker_board import (
+        initialize_breaker_board,
+    )
+
+    breaker_board = initialize_breaker_board()
```

Comment:
Instead of adding this comment, consider moving the initialization to a dedicated method elsewhere.